recipes/doxygen/all/conandata.yml

```diff
@@ -18,6 +18,10 @@ sources:
     url: "https://sourceforge.net/projects/doxygen/files/doxygen-1.8.17.src.tar.gz"
     sha256: "2cba988af2d495541cbbe5541b3bee0ee11144dcb23a81eada19f5501fd8b599"
 patches:
+  "1.9.4":
+    - patch_file: "patches/1.9.4-0001-enable-modern-compilers.patch"
+      patch_description: "Enable modern compilers"
+      patch_type: "bugfix"
```

Comment:
```suggestion
      patch_type: "portability"
```

Reply:
my read of https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conandata_yml_format.md#L209 is that that `bugfix` would be more appropriate than `portability`

Reply:
bugfix is for bugs in code leading to runtime errors, while portability is everything related to build itself, that's my understanding.

Reply:
Yeah I can squint at that interpretation.. aside from the build/runtime consideration, this description is the closest fit:
> Indicates a patch that backports an existing bug fix from the newer release or master branch (or equivalent, such as main/develop/trunk/etc).
@prince-chrismc @uilianries can either of you provide clarity on what patch type is most appropriate when a later bug fix affects a build?

Reply:
Bugfix == library code implementation 
Portability == new platforms or build environments

Reply:
Implementation... you are literally adding a header, that won't change the logic in any way.

It's portability here